### ISSUE TITLE
How to consume Ulule’s Webhook 

### ISSUE THREAD

**Issue by bmenant** [ulule/developers.ulule.com#20]
Is there an API endpoint to register a URL to deliver webhook requests to?

**Comment by yansal** [comment]
No, at that time the only way is to contact a human at api@ulule.com

This feature may be added in the future, we'll update this issue if that's the case.
